uber/decorators.py

```diff
@@ -513,7 +513,7 @@ def render(template_name_list, data=None, encoding='utf-8'):
 
 def render_empty(template_name_list):
     env = JinjaEnv.env()
-    template = env.get_or_select_template(template_name_list)
+    template = env.get_or_select_template(template_name_list, use_request_cache=False)
```

Comment:
This is the key line. What was happening is that whenever there were _multiple_ automated emails that used the same email template, each time they used a subsequent version of the template from different plugins. 

For example, these two emails both used `attendee_confirmation.html`:
```
AutomatedEmailFixture(
    Attendee,
    '{EVENT_NAME} payment received',
    'reg_workflow/attendee_confirmation.html',
    lambda a: a.paid == c.HAS_PAID,
    needs_approval=False,
    allow_at_the_con=True,
    ident='attendee_payment_received')

AutomatedEmailFixture(
    Attendee,
    '{EVENT_NAME} registration confirmed',
    'reg_workflow/attendee_confirmation.html',
    lambda a: a.paid == c.NEED_NOT_PAY and (a.confirmed or a.promo_code_id),
    needs_approval=False,
    allow_at_the_con=True,
    ident='attendee_badge_confirmed')
```

The first one would get `magstock/templates/emails/reg_workflow/attendee_confirmation.html` and the second one would get `uber/templates/emails/reg_workflow/attendee_confirmation.html` because the jinja environment would overlook any templates already loaded in the request cache.